tests/test_parse_enex.py

```
@@ -0,0 +1,472 @@
+        assert p.exists()
+
+
+# ---------- エッジケース ----------
+
+
+def _make_minimal_note(
+    title: str,
+    created: str = "20260428T094931Z",
+    body: str = "<en-note></en-note>",
+    tags: list[str] | None = None,
+    with_resource: bool = False,
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)